docs/init.md

```diff
@@ -66,7 +66,7 @@ react-native init ProjectName
 
 # Creating custom template
 
-Every custom template needs to have configuration file called `template.config.js` in the root of the project:
```

Comment:
Um, why this change? We resolve path to `template.config` but then using Node require resolution:

https://github.com/react-native-community/cli/blob/f316f9269bdd71d492efd8b73104f38c568c9e52/packages/cli/src/commands/init/template.js#L31-L40

Reply:
True, seems like I did a mistake yesterday while creating a new project.